mappings/net/minecraft/item/ItemStack.mapping

```diff
@@ -136,6 +149,14 @@ CLASS net/minecraft/unmapped/C_sddaxwyk net/minecraft/item/ItemStack
 		ARG 2 user
 		ARG 3 remainingUseTicks
 	METHOD m_iculdxhv isDamageable ()Z
+	METHOD m_igbhhnoy copyComponentsToNewStack (Lnet/minecraft/unmapped/C_gmbqjnle;I)Lnet/minecraft/unmapped/C_sddaxwyk;
+		COMMENT {@return a new item stack with the components copied from this stack, unless this stack is empty, then returns an empty item stack}
+		COMMENT
+		COMMENT @see #copyComponentsToNewStackIgnoreEmpty
```

Comment:
I'm assuming this should match the name for `m_cgmunnul`, which currently has an ambiguous `@see` tag which may reference the method itself

Reply:
I think I've fixed what you are talking about? I've made the `@see` tag on `m_cgmunnul` specifically reference the correct overloaded method.